src/components/Dropdown/Dropdown.module.scss

```diff
@@ -8,7 +8,7 @@
 .content_wrapper {
   border-radius: 4px;
   z-index: 1;
-  background-color: $white;
+  background-color: $color-background-neutral-0;
```

Comment:
this is changed because I was deleting a bunch of unused colors.. there shouldn't be a "white"in here..